src/styles/transitions.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
```

Comment:
Consider using nested selectors to organize the code and make it more readable. Also, make sure to use underscores to separate block and element names according to the task requirements. For example, you could group the `.item` and `.temp-item` related classes with nested selectors: